src/main/java/com/maxmind/minfraud/response/InsightsResponse.java

```diff
@@ -72,6 +94,7 @@ public String toString() {
         sb.append(", warnings=").append(this.warnings);
```

Comment:
You didn't add `device` to the `toString()`. That said, for GeoIP2, I replaced the `toString()` methods with [output created from the JSON](https://github.com/maxmind/GeoIP2-java/commit/7706e126a6768a76e76191f36e18f454a8ce329c). We could do that for these model classes too. `toString()` is used primarily as a debugging aid for programmers.